aws/cloudformation-templates/template.yaml

```diff
@@ -211,6 +221,7 @@ Resources:
         PinpointAppId: !GetAtt Base.Outputs.PinpointAppId
         ParameterPersonalizeEventTrackerId: !GetAtt Base.Outputs.ParameterPersonalizeEventTrackerId
         ParameterAmplitudeApiKey: !GetAtt Base.Outputs.ParameterAmplitudeApiKey
+        ParameterOptimizelySdkKey: !GetAtt Base.Outputs.OptimizelySdkKey
```

Comment:
Should be `!GetAtt Base.Outputs.ParameterOptimizelySdkKey` rather than `!GetAtt Base.Outputs.OptimizelySdkKey`